README.md

```diff
@@ -104,11 +104,11 @@ You can also stay up to date by reading the [FreeTube Blog](https://write.as/fre
 
 ## Donate
 
-[FreeTube on Liberapay](https://liberapay.com/FreeTube)
+* [FreeTube on Liberapay](https://liberapay.com/FreeTube)
 
-Bitcoin Address: 1Lih7Ho5gnxb1CwPD4o59ss78pwo2T91eS
+* Bitcoin Address: `1Lih7Ho5gnxb1CwPD4o59ss78pwo2T91eS`
 
-Monero Address: 48WyAPdjwc6VokeXACxSZCFeKEXBiYPV6GjfvBsfg4CrUJ95LLCQSfpM9pvNKy5GE5H4hNaw99P8RZyzmaU9kb1pD7kzhCB
+* Monero Address: `48WyAPdjwc6VokeXACxSZCFeKEXBiYPV6GjfvBsfg4CrUJ95LLCQSfpM9pvNKy5GE5H4hNaw99P8RZyzmaU9kb1pD7kzhCB`
 
 If you enjoy using FreeTube, you're welcome to leave a donation using the following methods.  While your donations are much appreciated, only donate if you really want to.  Donations are used for keeping the website up and running and eventual code signing costs.
```

Comment:
I think its better to move this above the donate methods

Reply:
Maybe in another separated change will be better?

Reply:
Could u maybe provide a visual example of that so i can decide better on that

Reply:
@PeterDaveHello